tdesign/src/Dialog/Dialog.tsx

```diff
@@ -34,14 +36,16 @@ const Dialog = ({
       }}
       {...(rest as any)}
     >
-      <CloseableDialogTitle
-        id={title}
-        onClose={() => {
-          setShowDialog(false);
-        }}
-      >
-        {title}
-      </CloseableDialogTitle>
+      {showCloseIcon && (
```

Comment:
Nit: if I understand correctly, it seems like this prop shows/hides the entire title bar of the Dialog, not just the CloseIcon. Would we want to call it `showDialogTitle` maybe?